.devcontainer/cosmos-emulator/cosmos-emulator-init.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# -*- coding: utf-8 -*-
+import subprocess
+import sys
+
+from azure.cosmos import CosmosClient, PartitionKey, exceptions
+import argparse
+import urllib3
+from contextlib import contextmanager
+
+@contextmanager
+def disable_ssl_warnings():
+    import warnings
+    import urllib3
+
+    with warnings.catch_warnings():
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+        yield None
+
+"""
+The code below disables SSL Cert verification
+It's only meant to be used with Cosmos Emulator
+Careful when using it with Azure Cosmos in production
+"""  
+
+def arg_parser():
+    parser = argparse.ArgumentParser("cosmos-emulator-init")
+    parser.add_argument("--key", "-k", required=True)
+    parser.add_argument("--url", "-u", required=True)
+    parser.add_argument("--emulate","-e", default=False, action="store_true")
+    return parser
+
+args = arg_parser().parse_args()
+
+KEY=args.key
+URL=args.url
+connection_verify=not args.emulate
+
+# For local emulator
+if connection_verify is False:
+    
+    warning="BEAWARE: SSL verification and warnings are disabled for emulator"
+    print(warning,file=sys.stderr)
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+client = CosmosClient(URL, credential=KEY, connection_verify=connection_verify)
+
+# Create LodeRunner db and container
+db_cont = {"LodeRunnerDB": "LodeRunner", "LodeRunnerTestDB": "LodeRunner"}
```

Comment:
i would add a note to here and other files that control where collections are set to keep in sync. just to prevent tech debt in future

Reply:
good point. just added a note